### DISCUSSION TITLE
Unable to connect to the server

### DISCUSSION THREAD

**Discussion by fedorov** [Project-MONAI/MONAILabel#269] [Q&A] [answered]
I installed monailabel on a Google Cloud Ubuntu VM, and started the server as described in the documentation. I see that there is a listener on port 8000, but I am unable to connect to the server either via 127.0.0.1/8000 or 0.0.0.0/8000. Any suggestions what I may be doing wrong?

![image](https://user-images.githubusercontent.com/313942/129234298-172e3be0-cf2a-41d6-9b19-219f7f570f43.png)

**Comment by SachidanandAlle** [comment]
Have you tried http://127.0.0.1:8000/ or http://127.0.0.1:8000/info/ in browser.. ? Or wget/curl on the same host where you run the server

**Comment by RafaelPalomar** [comment, marked as answer]
Hello @fedorov. In case it helps: I was failing to connect to `localhost:8000` from the 3D Slicer extension. The error is that I needed to specify a full valid URL  (e.g., `http://localhost:8000`).
